libibumad/man/umad_get_ca_device_list.3.md

```diff
@@ -0,0 +1,63 @@
+
+---
+date: "May 1, 2018"
+footer: "OpenIB"
+header: "OpenIB Programmer's Manual"
+layout: page
+license: 'Licensed under the OpenIB.org BSD license (FreeBSD Variant) - See COPYING.md'
+section: 3
+title: UMAD_GET_CA_DEVICE_LIST
+---
+
+# NAME
+
+umad_get_ca_device_list - get list of available InfiniBand device names.
+
+# SYNOPSIS
+
+```c
+#include <infiniband/umad.h>
+
+struct umad_device_node *umad_get_ca_device_list(void);
+```
+
+# DESCRIPTION
+
+**umad_get_ca_device_list()**  fills the cas list of *struct umad_device_node*
+with local IB devices (CAs) names.
+
+*struct umad_device_node* is defined as follows:
+
+```c
+struct umad_device_node {
+	struct umad_device_node *next; /* next umad device node  */
+	const char *ca_name;                 /* ca name  */
```

Comment:
No trailing comments in man pages

Write proper definitions lists, see something like the esp md examples